inventory/byo/hosts.ose.example

```diff
@@ -363,6 +363,8 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 #
 # Override metricsPublicURL in the master config for cluster metrics
 # Defaults to https://hawkular-metrics.{{openshift_master_default_subdomain}}/hawkular/metrics
+# Currently, you may only alter the hostname portion of the url, alterting the
+# `/hawkular/metrics` path will break installation of metrics.
 #openshift_hosted_metrics_public_url=https://hawkular-metrics.example.com/hawkular/metrics
```

Comment:
p.s. 
found where they forgot to set the routes path, it's in origin-metrics ( it's not an openshift-ansible thing ) :
[they set the `--hostaname=..` but leave the `--path` missing ... ]
https://github.com/openshift/origin-metrics/blob/master/deployer/scripts/hawkular.sh#L198